job-service-acceptance-tests/src/test/java/com/hpe/caf/jobservice/acceptance/JobServiceEndToEndIT.java

```diff
@@ -246,7 +246,10 @@ public void testTargetPipeForJobWithNoAndWithCompletedPrerequisiteJobs(final Str
         // prereq job ids that should not hold the job back.
         createJobWithPrerequisites(job3Id, true, 0, job1Id, job2Id, "", null, "           ");
 
-        Thread.sleep(1000); // Add short delay to allow previous job to complete
+        Thread.sleep(4000); // Add short delay to allow previous job to complete
```

Comment:
Let's see if adding an appropriate index to the `job_dependency` table will eliminate the requirement to increase these times.